### PR TITLE
Processor Basic Info Endpoint. HasBeenConfigured Config

### PR DESCRIPTION
**Pull request by renzodgc**
- Added an endpoint `GET /config/info` that returns some basic info and processor's version.
- Added a new config under `App`, `HasBeenConfigured` that is False by default and updates to  True if the config was updated.
- Modified README to let users know that if they manually change the setup they should also change this config.
- Added a constants.py file
- Added a .env and .env.example (moved email there)

### CODE REVIEW

File: api/config.py [hunk 1]
```diff
@@ -33,6 +49,18 @@ def map_config(config, options):
     }
 
 
+def processor_info(config):
+    has_been_configured = bool(config["App"]["HasBeenConfigured"])
+    device = config["Detector"]["Device"]
+    if config["Detector"]["Name"] == "openvino":
+        device += "-openvino"
+    return {
+        "version": "0.2.0",
```
Comment (pgrill) [inline api/config.py:58]: Can we move the version number to an independent file?
It should be in a place easy to find and edit each time we deploy a new one.

**Review by pgrill — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -212,12 +212,16 @@ You can read and modify the configurations in `config-*.ini` files, accordingly:
...
+Please note that if you modify these values you should also set `[App]` `HasBeenConfigured` to `"True"`.
+This allows for a client to recognize if this processor was previously configured.
+
 You can also modify some of them using the [UI](https://beta.lanthorn.ai). 
 If you choose this option, make sure to mount the config file as a volume to keep the changes after any restart of the container.
 
 All the configurations are grouped in *sections* and some of them can vary depending on the chosen device.
 
 - `[App]`
+  - `HasBeenConfigured`: A boolean parameter that state if the *config.ini* was changed or not.
```
Comment (mats-claassen) [inline README.md:224]:
```suggestion
  - `HasBeenConfigured`: A boolean parameter that states whether the *config.ini* was set up or not.
```

File: constants.py [hunk 3]
```diff
@@ -0,0 +1 @@
+PROCESSOR_VERSION = "0.2.0"
```
Comment (mats-claassen) [inline constants.py:1]: I think this should be set when a release is made, but let's suppose we will create it soon

**Review by mats-claassen — APPROVED** [review]

File: constants.py (hunk 3 above)
Comment (mats-claassen) [reply constants.py:1]: I mean that technically we are still 0.1.0

Comment (renzodgc) [reply constants.py:1]: That's true, we could include the changelog in this pull request

Comment (renzodgc) [inline constants.py:1]:
```suggestion
PROCESSOR_VERSION = "0.1.0"
```

**Pull request merged**